tests/Tests.dfy

```diff
@@ -13,30 +13,23 @@ module Tests {
   import DiscreteLaplace
   import DiscreteGaussian
   import NatArith
+  import RealArith
+  import Permutations
+  import FisherYates
+  import Helper
 
-  function Abs(x: real): real {
-    if x < 0.0 then -x else x
-  }
-
-  function NatToString(n: nat): string {
-    match n
-    case 0 => "0" case 1 => "1" case 2 => "2" case 3 => "3" case 4 => "4"
-    case 5 => "5" case 6 => "6" case 7 => "7" case 8 => "8" case 9 => "9"
-    case _ => NatToString(n / 10) + NatToString(n % 10)
-  }
-
-  method TestBernoulliIsWithin4SigmaOfTrueMean(
+  method TestBernoulliIsWithin3SigmaOfTrueMean(
     n: nat,
     empiricalSum: real,
     successProb: real,
     description: string
   )
     requires n > 0
   {
-    TestEmpiricalIsWithin4SigmaOfTrueMean(n, empiricalSum, successProb, successProb * (1.0 - successProb), description);
+    TestEmpiricalIsWithin3SigmaOfTrueMean(n, empiricalSum, successProb, successProb * (1.0 - successProb), description);
   }
 
-  method TestEmpiricalIsWithin4SigmaOfTrueMean(
+  method TestEmpiricalIsWithin3SigmaOfTrueMean(
```

Comment:
Just FYI: we had a threshold of 3 sigma before and quite regularly got spurious test failures with it.